workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/public-sites-navigation/index.js

```diff
@@ -308,13 +308,17 @@ window.addEventListener('keyup', (event) => {
 		siteSearchWrapper.classList.remove('search-open');
 	}
 
-	if (event.code === 'Slash' || event.key === '/') {
+	if (
```

Comment:
@luisacmn instead of just fixing this issue for the search bar, I think it may make sense to add an if check that returns out of the function if the element you are in is an input. Do you think that would work?

Reply:
hey ryan, sure ! I'll take a look!